web/src/components/core/Page.jsx

```diff
@@ -94,6 +95,30 @@ const BackAction = () => {
   );
 };
 
+/**
+ * Internal component for displaying a button to show IssuesDialog when there
+ * are issues
+ *
+ * FIXME: move to its own component and import it instead (?)
```

Comment:
But maybe it should follow a similar approach as Popup and Page. I.e., to move it to `IssuesDialog` component and have a `<IssuesDialog.OpenButton />` or so.

@imobachgs, what do you think about it? Just looking for another view before going ahead.